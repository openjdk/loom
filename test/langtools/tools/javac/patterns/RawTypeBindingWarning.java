/*
 * @test /nodynamiccopyright/
 * @bug 8263590
 * @summary Verify correct warnings are produced for raw types in bindings
 * @compile/ref=RawTypeBindingWarning.out -Xlint:rawtypes -XDrawDiagnostics RawTypeBindingWarning.java
 */
public class RawTypeBindingWarning<T> {
    public static boolean t(Object o) {
        return o instanceof RawTypeBindingWarning w;
    }
}
