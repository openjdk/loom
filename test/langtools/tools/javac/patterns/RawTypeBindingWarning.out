RawTypeBindingWarning.java:9:29: compiler.warn.raw.class.use: RawTypeBindingWarning, RawTypeBindingWarning<T>
1 warning
